sdk/tables/azure-data-tables/migration_guide.md

```diff
@@ -0,0 +1,289 @@
+- Using the new `azure-identity` library to share a single authentication approach between clients -->
+- A unified logging and diagnostics pipeline offering a common view of the activities across each of the client libraries
+
+### Performance improvements
+
+Use this section to advertise the performance improvements in new package when compared to the old one. Skip this section if no perf improvements are found yet.
+
+### New features
+
+We have a variety of new features available in the new library:
+* Ability to submit a batch of operations with the `TableClient.submit_transaction` method
+
+## Important changes
+
+### Package names and namespaces
+
+The package name has been changed from `azure-cosmosdb-table` to `azure-data-tables`. This package can target either CosmosDB or Azure Storage Tables accounts.
+
+### Client hierarchy and constructors
+
+In the interest of simplicity, there are only two clients, `TableServiceClient` for account-level interactions and `TableClient` for table-level interactions. This is in contrast to the single `CosmosClient` used for interacting with the account level operations and the proxies used for interactions with specific resources (ie. `DatabaseProxy` and `ContainerProxy`).
+
+### Authenticating Clients
+
+In `azure-cosmosdb-table`:
+```python
+import os
+from azure.cosmosdb.table import TableService
+
+client = TableService(account_name=os.environ["ACCOUNT_URI"], account_key=os.environ["ACCOUNT_KEY"])
+```
+
+In `azure-data-tables`:
+```python
+import os
+from azure.core.credentials import AzureNamedKeyCredential
+from azure.data.tables import TableClient, TableServiceClient
+
+key = os.environ["TABLES_ACCOUNT_KEY"]
+name = os.environ["TABLES_ACCOUNT_NAME"]
+url = os.environ["TABLES_ACCOUNT_URL"]
+
+credential = AzureNamedKeyCredential(key=key, name=name)
+
+table_client = TableClient(account_url, "tablename", credential=credential)
+
+table_service_client = TableServiceClient(account_url, credential=credential)
+```
+
+
+### Table Level Scenarios
+
+#### Create and Delete
+
+In `azure-cosmosdb-table`:
+```python
+from azure.cosmosdb.table import TableService
+client = TableService(...)
+table_name = "tableName"
+created = client.create_table(table_name)
+# create_table returns True if a new table was created, False if the table already exists
+
+# Delete
+table_name = "deleteTableName"
+client.delete_table(table_name)
+# delete_table returns True if a table was deleted, False if the table does not exist
+```
+
+In `azure-data-tables`:
+```python
+from azure.data.tables import TableServiceClient, TableClient
+
+# You can create a table from the TableServiceClient or TableClient
+service_client = TableServiceClient.from_connection_string(conn_str)
+table_client = service_client.create_table("tableName")
+service_client.delete_table("tableName")
+
+# The create_table_if_not_exists
+table_client = TableClient.from_connection_string(conn_str, table_name="tableName")
```

Comment:
for clarity, what is the difference b/w using both clients to create a table. Why can both be used to do this?

Maybe add that explanation at the bottom or top of this section (### Create and delete)